source/MonoGame.Extended.Input/InputListeners/GamePadListener.cs

```diff
@@ -71,7 +71,7 @@ public GamePadListener(GamePadListenerSettings settings)
             RepeatDelay = settings.RepeatDelay;
 
             _previousGameTime = new GameTime();
-            _previousState = GamePadState.Default;
+            _previousState = new GamePadState();
```

Comment:
GamePadState.Default is an MG extension that's wrapping new GamePadState(). It doesn't exist in FNA.